ab_tool/templates/ab_tool/editExperiment.html

```diff
@@ -173,7 +179,8 @@ <h4 class="modal-title">Edit Summary</h4>
 <script type="text/javascript">
     {% if create %}
         window.initialExperiment = {"name": "", "notes": "", "uniformRandom": true,
-            "tracks": [{"id": null, "name": "A"}, {"id": null, "name": "B"}]};
+        "tracks": [{"id": null, "name": "A", "newName": "A"},
```

Comment:
Don't you need to define `originalName` here, in addition to the other name keys?